### DISCUSSION TITLE
Atomic charges questions

### DISCUSSION THREAD

**Discussion by Liaokang123456** [CederGroupHub/chgnet#178] [Q&A] [answered]
Hi, I read your paper ["CHGNET"](https://doi.org/10.48550/arXiv.2302.14231), and I got confusions and have some questions, You use magnetic moments to represent the atomic charges, and train the model by using its, and the informations need DFT calculations, So my question is when we predict the E,F,S,M of one structure, we have to do DFT calculations before?
Or how to define the atomic charge informations in training? 
Thanks !

**Comment by bowen-bd** [comment, marked as answer]
Hi there!

You don't have to do DFT calculation.
Atomic charges can be inferred from electron orbital occupancy, as unpaired electron contributes to spin.
The charge inference from spin(magmom) is a widely used technique especially in electro chemistry related papers using DFT.

The inference rule depends on the element you're using, and if you're unsure about that, I recommend look for similar compounds inside [Materials Project](https://materialsproject.org), and you can check the valence states and magmoms that are calculated directly from DFT.
